NDB.Covid19/NDB.Covid19/Locales/nb.json

```diff
@@ -248,6 +248,7 @@
   "INFECTION_STATUS_BACKGROUND_ACTIVITY_DIALOG_MESSAGE_PART3": "Tillater du å slå den på?",
   "INFECTION_STATUS_SURVEY_HEADER_TEXT": "Spørreundersøkelse for deg som har testet deg",
   "INFECTION_STATUS_SURVEY_LINK_URL": "https://nettskjema.no/a/195329",
+  "INFECTION_STATUS_INFORMATION_BANNER_LINK_URL": "https://www.helsenorge.no/",
```

Comment:
I think maybe it makes sense to delete this at all? This change will either way be overwritten in the future, so maybe set the link currently directly in the ViewModel

Reply:
okay i put it in the viewmodel instead.